netpyne/batch/grid.py

```diff
@@ -291,12 +291,11 @@ def gridSubmit(batch, pc, netParamsSavePath, jobName, simLabel, processes, proce
         print('Skipping job %s since cfg file already exists...' % (jobName))
     elif batch.runCfg.get('skipCustom', None) and glob.glob(jobName+batch.runCfg['skipCustom']):
         print('Skipping job %s since %s file already exists...' % (jobName, batch.runCfg['skipCustom']))
-    else:
```

Comment:
hey @adamjhn can you explain why you removed this code from the else block? thanks

Reply:
I suppose, what's missing in this code is `return` statement in each of `if` and `elif` blocks where we're skipping. And then indeed the last block of code can be moved out from `else` block. @salvadord